src/main/java/org/broadinstitute/hellbender/engine/spark/GATKSparkTool.java

```diff
@@ -57,13 +57,13 @@
     @ArgumentCollection
     protected IntervalArgumentCollection intervalArgumentCollection = requiresIntervals() ? new RequiredIntervalArgumentCollection() : new OptionalIntervalArgumentCollection();
 
-    private JavaSparkContext sparkContext;
-    private ReadsSparkSource readsSource;
-    private SAMFileHeader readsHeader;
+    private transient JavaSparkContext sparkContext;
+    private transient ReadsSparkSource readsSource;
+    private transient SAMFileHeader readsHeader;
     private String readInput;
-    private ReferenceMultiSource referenceSource;
-    private SAMSequenceDictionary referenceDictionary;
-    private List<SimpleInterval> intervals;
+    private transient ReferenceMultiSource referenceSource;
+    private transient SAMSequenceDictionary referenceDictionary;
+    private transient List<SimpleInterval> intervals;
```

Comment:
This change would mean tools would be in an inconsistent state after serialization, with many operations unusable (such as `getReads()` and `getHeaderForReads()`), wouldn't it?